code/go/0chain.net/chaincore/chain/state_handler.go

```diff
@@ -60,15 +60,20 @@ func SetupScRestApiHandlers() {
 }
 
 /*SetupStateHandlers - setup handlers to manage state */
-func SetupStateHandlers() {
+func SetupSharderStateHandlers() {
 	c := GetServerChain()
 	http.HandleFunc("/v1/client/get/balance", common.WithCORS(common.UserRateLimit(common.ToJSONResponse(c.GetBalanceHandler))))
-	http.HandleFunc("/v1/scstate/get", common.WithCORS(common.UserRateLimit(common.ToJSONResponse(c.GetNodeFromSCState))))
```

Comment:
This endpoint is required for both miners and sharders, but as @dabasov suggested, we can put it under `development` endpoints, so as the `config/get`.